### DISCUSSION TITLE
Ubuntu Live Server Support

### DISCUSSION THREAD

**Discussion by ProfessorManhattan** [antonym/netboot.xyz#706] [General] [answered]
Hey, are there any plans to include the new Ubuntu 20.04 Live Server install?

Or does anyone have any configs they can share? I'm interested in being able to have a completely automated install and I don't think the current Network Install has the possibility for it. I heard the new Live Server automatically generates a config based on your choices during the installation.. so that can be used for complete automation the second time around.

**Comment by antonym** [comment]
I hadn't heard of anything new, got a link?  

The network install can do it today, you just need to specify a preseed file either through the Ubuntu netboot.xyz menu or you can add custom kernel line options via the Utility menu.

**Comment by antonym** [comment, marked as answer]
Subiquity Live installers are now default for 20.04 LTS and 20.10.  You should be able to leverage autoinstall as well:

https://ubuntu.com/server/docs/install/autoinstall-reference

**Comment by ProfessorManhattan** [comment]
Awesome!
